### PR TITLE
feat(unlock-app): Add email option in paywall config for automatic collection of email

### PR DESCRIPTION
**Pull request by searchableguy**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Review by julien51 — CHANGES_REQUESTED** [review]
Just a few nits and then lg.
Please make sure to add the corresponding docs too

File: unlock-app/src/components/interface/checkout/alpha/Checkout/Metadata.tsx [hunk 1]
```diff
@@ -101,23 +101,25 @@ export function Metadata({ checkoutService, injectedProvider }: Props) {
...
+      return
+    }
 
-      const inputs: MetadataInput[] =
-        locksConfig.metadataInputs || paywallConfig.metadataInputs || []
+    const emailInput: MetadataInput = {
+      type: 'email',
+      name: 'email',
+      required: true,
+      placeholder: 'unlock-protocol@gmail.com',
```
Comment (julien51) [inline unlock-app/src/components/interface/checkout/alpha/Checkout/Metadata.tsx:112]:
```suggestion
      placeholder: 'hello@unlock-protocol.com',
```

**Review by julien51 — APPROVED** [review]

**Comment by julien51** [comment]
> Please make sure to add the corresponding docs too

Please @searchableguy make sure you add some docs on https://docs.unlock-protocol.com/tools/checkout/configuration

**Comment by searchableguy** [comment]
will do after merging

**Pull request merged**